sunpy/map/mapcube.py

```diff
@@ -139,11 +144,25 @@ def plot(self, gamma=None, axes=None, resample=None, annotate=True,
         >>> writer = Writer(fps=10, metadata=dict(artist='SunPy'), bitrate=1800)
 
         >>> ani.save('mapcube_animation.mp4', writer=writer)
+
+        Save an animation with the limb at each time step
+
+        >>> def myplot(fig, ax, map):
+        ...    p = map.draw_limb()
```

Comment:
we should make it clear for the example but, of course it doesn't actually matter, it's a positional argument the user can call it whatever they want.